updater/reports/repository/ReportGitRepositorySizeObjectsSize.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+from .ReportRepository import *
+
+class ReportGitRepositorySizeObjectsSize(ReportRepository):
+	def name(self):
+		return "git-repository-size-objects-size"
+
+	def convertBytesToMiB(self, bytes):
+		return round(bytes / 1024 ** 2, 3)
+
+	def updateData(self):
+		stdout = self.executeScript(["ghe-repo", self.repository, "-c", self.configuration["gitSizerPath"] + " --json --json-version=2"])
```

Comment:
That is nice! But don't we run `git-sizer` here twice per repo? Here and there: https://github.com/Autodesk/hubble/pull/177/files#diff-6b012ceb81adedd4a22e2089a0102dd3R10

Reply:
Yep, I'm aware and thought it's ok to optimize later. 🙂It would need a bit of refactoring I think, as the common design in the updater is to have one report file and class per report. But it shouldn't be too hard to find a solution for this.

Reply:
I see two options:

(1) All reports are executed sequentially. That means we could pass all reports a "cache dictionary" which is used to use data that a previous report calculated.

(2) We add a `cache` flag to `executeScript`. If this flag is set, then the output is cached and reused in a subsequent call with the same command.

@pluehne/@steffen Do you see another option? If no, which one would you prefer? I am leaning towards (2).

Reply:
@larsxschneider Thanks for these ideas! I consolidated the `git-sizer` reports into one: https://github.com/Autodesk/hubble/pull/177/commits/7008ec9662a4ed1d24d63683c50898f7061c4d32#diff-66c529dd9266003f0fc5892af8ac0cb8 

All quantity values are now tracked in one report. This gives us more flexibility on what to show while having all the data as discussed [here](https://github.com/Autodesk/hubble/pull/177#discussion_r272942449). I didn't really think of this idea of using the same report for different charts until I worked on the `git-download` reports which does exactly that. 🙂